tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -42,3 +45,40 @@ Feature: spaces management
 #      | id     |
 #      | team.b |
 #    And "Brian" logs out
+
+
+  Scenario: multiple spaces can be managed at once in the admin settings via the batch actions
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+    And "Admin" assigns following roles to the users
+      | id    | role       |
+      | Alice | SpaceAdmin |
+    When "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" navigates to the projects space page
+    And "Alice" creates the following project spaces
```

Comment:
There is an API step by now for creating project spaces: `creates the following project space(s) using API`. I think for the preparation of the test background / context it's preferable to use that API step.